_pages/en/building-enabling-directorate.md

```diff
@@ -0,0 +1,82 @@
+
+###### Time Management
+
+Teams should not be fully allocated to their enabling function.
+
+Firstly, each team must resolve time for the continuous self-improvement, which should be equal to, or strictly greater than, 20% of their teams available work time.
+
+Secondly, teams are free to self organize under the condition that they are able to meet the needs of the directorate’s priorities. For example, a team within the directorate may accept to fulfill an enabling function for another team within the organization.
+
+Thirdly, an enabling team need not commit all of their time to being an enabling team. For example, in the case of the IT Strategy team they are also responsible for setting the IT vision for the department, and thus cannot dedicate 100%(-20%) of their time to their enabling function. In this particular case, for example, we recommend that the IT Strategy spend a maximum of (100-20)/2 percent of their time on enabling other teams.
+
+##### External Interactions
+
+This is an initial proposal and is intended to allow for iterations to refine and modify the process as needed.
+
+###### Interactions
+
+The book Team Topologies, referenced below, recommends that Enabling Teams work with other teams for a period (potentially a few months) [TO-DO: Quote needed] in order to assist them with onboarding new technologies and approaches, and improving their existing processes and procedures. The goal is to ensure the interactions are brief enough that the teams being enabled do not become dependent on the enabling teams. The process improvements should stand on their own, and should not involve [toil](https://landing.google.com/sre/sre-book/chapters/eliminating-toil/). That is to say, the new processes and procedures should aim to automate or improve the existing processes, not lend temporary person-power to the teams. Given the short and iterative nature of the interactions between the teams (duration of interactions should be defined by number of sprints) the enabling teams must identify quick wins -- the amount of automation that could be done within the limited time frame that would result in the greatest amount of automated work.
+
+###### Tooling
+
+[TO-DO: Quote from Team Topologies regarding varying levels of team communication]
+
+The amount of collaboration between the enabling teams and the enabled teams should vary significantly during the time which the enabling team is paired with the enabled team. As such, collaboration tools that allow for the rapid creation and disintegration should be leveraged. An example of a viable tool for such a requirement would be Slack, where groups can be created and disposed of with relative ease.
+
+#### Process
+
+##### Teams Iterative Process
+
+- All teams within the directorate have synchronized sprints
```

Comment:
That seems prescriptive. Why not let different teams use their own cadence?

Reply:
Hmm, unless it's planning at the directorate level? Not sure about this here.

Reply:
Because for the sake of coordinating work between teams, it would be best if they were all working on the same coordinated tasks at the same time. 

If work comes down from the director to the managers, if ones team sprint planning just happened, and the work was intended to take only one sprint, that means if the sprints are malaligned they will not be able to accomodate the work without having a mid-sprint planning session.

TLDR; it is to enable easier coordination between the teams under the same director.